content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -37,7 +37,16 @@ by providing the following flags to the kube-apiserver:
 1. Create an egress configuration file such as `admin/konnectivity/egress-selector-configuration.yaml`.
 1. Set the `--egress-selector-config-file` flag of the API Server to the path of
 your API Server egress configuration file.
-
+1. If you use UDS connection, add volumes config to the kube-apiserver:
+   ```
+     - mountPath: /etc/kubernetes/konnectivity-server
+       name: konnectivity
+   - hostPath:
+       path: /etc/kubernetes/konnectivity-server
+       type: DirectoryOrCreate
+     name: konnectivity
+   ```
```

Comment:
This indentation is weird. please double check.

Reply:
Done.